.merlin

```diff
@@ -1,56 +0,0 @@
-S ./asmcomp
```

Comment:
That was a mistake (thanks for having spotted it!) which has been fixed
by editing the commit. So the branch needs to be checked out again.